reports/blueprint/report.tex

```diff
@@ -46,10 +46,22 @@
 
     \end{center}
 \end{titlepage}
-\section*{Executive Summary}
-% pop a one pager in here. Ideally less
-% Timothy
-\newpage
+\section{Executive Summary}
+
+This report outlines TeachEE, a device which facilitates online learning for
+electrical engineering labs. It describes the project's scope: a minimum viable
+product for the purpose of this capstone project, with room for scope expansion
+later in the year if circumstances allow it. The minimum viable product consists
+of an oscilloscope, desktop app, and communication between a computer and the
+device via USB. A complete list of specifications for the MVP are provided in
+Table \ref{hw:specs-table} and Table \ref{sw:specs-table}.
+
+It also outlines a road-map for this project; most hardware related tasks have
+been completed, with the software related tasks scheduled for completion later
+in the term. Although the PCB shipment from the manufacturer was delayed, the
+project is still on schedule. Additionally, for the upcoming tasks, there is a
+clear and equal division of responsibility between group members, as shown in
+Table \ref{hw:milestones-table}.
```

Comment:
I would add a sentence saying that the project is targeted for completion in week 20, two weeks before the Open House, to allow for extra time in case of emergencies

Reply:
Also squash your commits into one with a meaningful message

Reply:
added da4b74cc7f090da9fb03cbb6443b190b9a9830e6